src/ludamus/client/src/index.css

```
@@ -393,66 +404,68 @@ button:active {
+     is no surrounding ring — that read as a grey line on light backgrounds. */
   .btn-primary {
-    /* Bright coral face via a subtle vertical gradient — lighter top, deeper
-       bottom — both stops are --color-primary (#f85a3c) nudged in lightness
-       (no color-mix: legacy support). --bg is the bottom stop so the 3D lip
-       matches the gradient base. The white label keeps the brand's bright coral
-       in both themes; the touches below fix dark-mode halation perceptually
-       rather than by recolouring. */
     --bg: #ef4f2d;
+    --hl1: rgba(255, 255, 255, 0.3);
+    --hl2: rgba(255, 255, 255, 0.22);
     color: white;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)